src/app.rs

```diff
@@ -175,6 +177,42 @@ pub struct Config {
     pub using_table: Option<String>,
     pub using_port: Option<u32>,
     // pub cache_expiration_time: Option<i64>, // in second. default 300 (= 5 minutes)
+    #[serde(default)]
+    pub retry_batch_write_item: RetryConfig,
```

Comment:
Could you introduce a configuration structure? I want to specify it as follows:

```yaml
---
using_region: local
using_table: Thread
using_port: 8000
retry:
  default:
    enabled: true
    max_attempts: 10
    max_backoff: 5
  batch_write_item:
    max_attempts: 20
    max_backoff: 180
```

If we do not specify the option in `batch_write_item`, we use the value from the `default`. If we do not specify both of them, we use the default behavior of dynein.

Reply:
I have corrected it as you suggested.